pkgs/development/libraries/libxmlb/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, meson, ninja, pkgconfig, glib, libuuid, gobject-introspection, gtk-doc, shared-mime-info, python3, docbook_xsl, docbook_xml_dtd_43 }:
+
+stdenv.mkDerivation rec {
+  name = "libxmlb-${version}";
+  version = "0.1.5";
+
+  outputs = [ "out" "lib" "dev" "devdoc" ];
+
+  src = fetchFromGitHub {
+    owner = "hughsie";
+    repo = "libxmlb";
+    rev = version;
+    sha256 = "037j9fwkzsy3765gl2grkrmbxrfs67wlai213qbgsa5xn6fb8y68";
+  };
+
+  nativeBuildInputs = [ meson ninja pkgconfig gobject-introspection gtk-doc shared-mime-info docbook_xsl docbook_xml_dtd_43 ];
+
+  buildInputs = [ glib libuuid python3 ];
```

Comment:
I think `python3` here is probably only needed during the build.

Reply:
Sounds good.